data/templates/csharp.cs

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env scriptisto
+
+using System;
+
+// scriptisto-begin
+// script_src: csharp.cs
+// build_once_cmd: dotnet new console --name csharp --force
+// # TODO: this works but could use refactoring
+// build_cmd: "SCRIPT_NAME=csharp && echo current_directory: $(pwd) && cp $SCRIPT_NAME.cs $SCRIPT_NAME/Program.cs && cd $SCRIPT_NAME && dotnet build --configuration Release"
+// # Change to dotnet version your system uses: net6.0, net7.0, net8.0
+// target_bin: ./csharp/bin/Release/net8.0/csharp
```

Comment:
Wouldn't it be possible to simlink it or copy it as a part of the build command to a fixed name?

E.g. "cp ./csharp/bin/Release/*/csharp ./csharp/bin/Release/binary"